CPAC/pipeline/schema.py

```diff
@@ -1001,15 +1009,19 @@ def sanitize(filename):
     },
     'regional_homogeneity': {
         'run': bool,
+        #'target_space': In({'Template', 'Native'}),
+        'target_space': [In(valid_options['regional_homogeneity']['target_space'])],
```

Comment:
I noticed these checks https://github.com/FCP-INDI/C-PAC/blob/e81170fffbedfd11a1c04c0f402a1ad052e923c2/CPAC/pipeline/cpac_pipeline.py#L1365-L1379 are all `if space in config` and that the schema sets these values as lists. Is this forkable? Like I can choose both and get outputs for each?

Reply:
Yes its forkable